api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019Report.java

```diff
@@ -114,9 +116,11 @@ public ReportDefinition constructReportDefinition() {
         CohortIndicatorDataSetDefinition dsd = new CohortIndicatorDataSetDefinition();
 
         dsd.setParameters(getParameters());
-        rd.addDataSetDefinition("TX_CURR", Mapped.mapStraightThrough(dsd));
+        rd.addDataSetDefinition("TX", Mapped.mapStraightThrough(dsd));
+        rd.addDataSetDefinition("S",Mapped.mapStraightThrough(settings()));
+        rd.addDataSetDefinition("aijar", Mapped.mapStraightThrough(getUgandaEMRVersion()));
 
-        CohortDefinitionDimension ageDimension =commonDimensionLibrary.getTxNewAgeGenderGroup();
+        CohortDefinitionDimension ageDimension =commonDimensionLibrary.getNewTxCurrAgeGenderGroup();
```

Comment:
@dbaluku Why not just update the contents of the changed method instead of creating a new one